### PR TITLE
New/debugging intellij

### PR DESCRIPTION
**Pull request by rlebre**
Added section to aid users to debug using IntelliJ IDE

I changed the showing of a section that was similar to the both methods (VSCode and IntelliJ) and split the instructions in two, keeping the template with the same subsections

### CODE REVIEW

File: _docs/debugging.md [hunk 1]
```diff
@@ -113,12 +107,22 @@ git clone https://github.com/bioinformatics-ua/dicoogle.git
 In this situation, you may consider checking out a released version to ensure compatibility:
 
 ```sh
-git checkout v2.5.0
+git checkout 2.5.0
```
Comment (Enet4) [inline _docs/debugging.md:110]: The latest v2 version is 2.5.4.

```suggestion
git checkout 2.5.4
```

File: _docs/debugging.md [hunk 2]
```diff
@@ -113,12 +107,22 @@ git clone https://github.com/bioinformatics-ua/dicoogle.git
...
+
+### Visual Studio Code
+
+#### Preparing the IDE
+
+You can download [Visual Studio Code](https://code.visualstudio.com) for free from the official website. Please ensure that you have the latest stable version. In order to debug Java programs, we also need the ["Java Extension Pack"](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which can be installed directly from the IDE, in the Extensions Marketplace. Please install this extension pack and reload Visual Studio Code afterwards.
+
+![The Java Extension Pack from the Extension Marketplace.]({{ site.baseurl }}/images/screenshot_debug_extension.png)
+
+
```
Comment (Enet4) [inline _docs/debugging.md:124]:
```suggestion

```

File: _docs/debugging.md [hunk 3]
```diff
@@ -173,3 +177,63 @@ We can now press the "Start Debugging" button on the Debug tab to run Dicoogle t
 ![Finally!]({{ site.baseurl }}/images/screenshot_debug_finally.png)
 
 It is now possible to slowly step through the code and observe the state of your plugin. More information on debugging with Visual Studio Code is available at the official website [here](https://code.visualstudio.com/docs/editor/debugging).
+
+
+
```
Comment (Enet4) [inline _docs/debugging.md:182]:
```suggestion

```

File: _docs/debugging.md [hunk 4]
```diff
@@ -173,3 +177,63 @@ We can now press the "Start Debugging" button on the Debug tab to run Dicoogle t
...
+ |   └── list-plugin.jar
+ ├── storage                (optional)
+ |   ├── «my-dicom-data»
+ |   └── ...
+ └── dicoogle.jar
+```
+
+Basically, we are preparing a working directory for debugging purposes, containing only the necessary plugins (don't forget all the plugins!). If some data is needed, include it as well.
+
+
```
Comment (Enet4) [inline _docs/debugging.md:208]:
```suggestion

```

File: _docs/debugging.md [hunk 5]
```diff
@@ -173,3 +177,63 @@ We can now press the "Start Debugging" button on the Debug tab to run Dicoogle t
...
+Open the source code's root folder with IntelliJ IDEA. 
+
+![Opening source folder]({{ site.baseurl }}/images/screenshot_debug_open_folder.png)
+
+Next, we will configure the Remote Debug in IntelliJ. On the top right corner, open "Add Configuration". From the "+" icon, choose "Remote", as shown in the screenshot below. You can use the default settings from the IntelliJ IDEA Remote run/debug template.
+
+![Add remote configuration]({{ site.baseurl }}/images/screeshot_add_configuration_intellij.png)
+
+
```
Comment (Enet4) [inline _docs/debugging.md:217]:
```suggestion

```

File: _docs/debugging.md [hunk 6]
```diff
@@ -173,3 +177,63 @@ We can now press the "Start Debugging" button on the Debug tab to run Dicoogle t
...
+To make sure that everything is properly set up, let's add a breakpoint or two in our code. In one the plugin set classes, add a breakpoint by clicking on the left side of a code line. A red dot should appear where you clicked.
+
+![Adding a breakpoint]({{ site.baseurl }}/images/screenshot_debug_breakpoint_intellij.png)
+
+We can now press the Debug button on top right corner (or by clicking F5) to start debugging. After a few moments, the program should stop at the given breakpoints.
+
+![Click debug]({{ site.baseurl }}/images/start_debugging_intellij.png)
+
+
```
Comment (Enet4) [inline _docs/debugging.md:234]:
```suggestion

```

**Review by Enet4 — COMMENTED** [review]
Only a few minor tweaks before this is merged.

File: _docs/debugging.md [hunk 7]
```diff
@@ -113,12 +107,20 @@ git clone https://github.com/bioinformatics-ua/dicoogle.git
 In this situation, you may consider checking out a released version to ensure compatibility:
 
 ```sh
-git checkout v2.5.0
+git checkout 2.5.4
```
Comment (Enet4) [inline _docs/debugging.md:110]: OK sorry, I've changed my mind a bit on this one. 😆 Let's recommend to use Dicoogle 3!
```suggestion
git checkout 3.0.2
```

Comment (rlebre) [reply _docs/debugging.md:110]: You are right! I thought you wanted to recommend the latest 2.5.x

**Review by Enet4 — APPROVED** [review]
Great!

**Pull request merged**